code/modules/surgery/surgery_helpers.dm

```diff
@@ -130,6 +130,16 @@
 	else
 		return 0.5
 
+/proc/get_internals_modifier(mob/M)
+	var/mob/living/carbon/human/H = M
+	if (!istype(H))
+		return 0.0
+	else if (istype(H.internal,/obj/item/tank/internals/anesthetic))

```

Comment:
And this allows for them to just fill it up with oxygen and release at 16 kpa, it'll count as being internal. They just won't be knocked out.

Reply:
Ah, absolutely. I will get working on implementing a check when I have some free time. People refilling anesthetic tanks is an unlikely scenario but it is important to cover all possibilities. I also just realized that this would work if the tank was empty. As I become more familiar with the byond environment and the tgstation code I should learn to avoid simple mistakes like this. Thank you for your feedback!

Reply:
replace with a sleeping/unconscious check? 

The tank should be giving you enough to go unconscious.